apps/sms/services/test/unit/messaging/messaging_service_test.js

```diff
@@ -51,7 +52,11 @@ suite('Messaging service >', function() {
   suite('init', function() {
     test('service mixing and client are initialized', function() {
       sinon.assert.calledWith(bridge.service, 'messaging-service');
-      sinon.assert.calledWith(bridge.client, 'mozMobileMessageShim');
+      sinon.assert.calledWith(bridge.client, {
+        service: 'moz-mobile-message-shim',
+        endpoint: sinon.match.instanceOf(BroadcastChannel),
```

Comment:
__optional nit:__ in my patch I've used slightly modified BroadcastChannelMock (mock remebers name that is passed into its constructor)with the following assert, to make sure we use channel with the right name:
```
endpoint: sinon.match.instanceOf(BroadcastChannel).and(sinon.match.has('name', shimChannel))
```

But up to you.